modules/dxp/apps/portal-workflow/portal-workflow-metrics-test/src/testFunctional/macros/WorkflowMetricsProcessListView.macro

```diff
@@ -10,4 +10,17 @@ definition {
 		Click(locator1 = "WorkflowMetricsProcessListView#METRICS_WORKFLOW_SEARCH_BAR_BUTTON");
 	}
 
+	macro metricsWorkflowSearchClearAll {
+		var key_workflowSearchName = "${workflowSearchName}";
+
+		Type(
+			key_fieldLabel = "Search for",
+			locator1 = "WorkflowMetricsProcessListView#METRICS_WORKFLOW_SEARCH_BAR",
+			value1 = "${workflowSearchName}");
+
+		Click(locator1 = "WorkflowMetricsProcessListView#METRICS_WORKFLOW_SEARCH_BAR_BUTTON");
+
+		Click(locator1 = "WorkflowMetrics#FILTER_CLEAR_ALL");
```

Comment:
Can you add an assert to assert that the clear was successful please?

Reply:
Hi @CalumR23 I did a verification where it says that the "Clear All" element is not present, this logic states that the clean was successful

Reply:
Awesome thank you!